trunk/SUAVE/Attributes/Propellants/Liquid_H2.py

```diff
@@ -1,50 +0,0 @@
-## @ingroup Attributes-Propellants

```

Comment:
Any reason why this was deleted?

Reply:
It wasn't used, and it was wrong.  will be re-add when scramjet is complete. #100percentcoverage

Reply:
it wasn't that wrong........but it wasn't used